indexes/plugin.js

```diff
@@ -40,7 +40,7 @@ module.exports = class Plugin {
     this.batch = []
 
     this.flush = (cb) => {
-      if (processedOffset < 0 || this.level.isClosed()) return cb()
+      if (processedOffset === this.offset.value || this.level.isClosed()) return cb()
```

Comment:
note this was also a bit wierd before because `processedOffset` might be set, but we didn't actually change anything.

Reply:
How about `if (processedOffset < 0 || processedOffset === this.offset.value || this.level.isClosed())` ? Because the initial value of `this.offset.value` is not -1 if I remember correctly.

Reply:
It should be -1 if the file does not exist so that extra check should not be needed I think